### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by RudraAsthana** [mbrubeck/agate#68] [Q&A]
Hey there can anybody help me with setting up a custom domain? I don't know exactly how to import it.

**Comment by Johann150** [comment]
Maybe you could be a bit more specific about what exactly you have an issue with? How is this related to agate?

**Reply by RudraAsthana** [reply]
Thanks, For addressing!

I was asking that when we open a server how would set a domain in it so that anybody can just type that domain name with Gemini protocol and get to my website.

If my question is irrelevant to agate. I am so sorry

**Reply by Johann150** [reply]
You will have to buy/register a domain at a domain registrar and set up DNS to point to your servers.

Sorry, but I can't help you with that. You will probably find plenty guides online. If you have set up a server so that DNS points to it, feel free to reach out again.

**Reply by RudraAsthana** [reply]
Thanks a lot! 
Have a great day ahead
